test/property.test.js

```diff
@@ -125,6 +207,20 @@ describe('loopback:property generator', function() {
     });
   });
 
+  it('should fail to create with unsupported type', function (done) {
+    var propertyGenerator = givenPropertyGenerator();
+    helpers.mockPrompt(propertyGenerator, {
+      model: 'Car',
+      name: 'something',
+      type: 'unknown',
```

Comment:
@bajtos, please correct me if I'm wrong but apic uses generator-loopback programatically doesn't it?

(if the answer is anything other than a simple "yes" or "no", let's take it offline)